net/amneziawg-tools/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2016-2019 Jason A. Donenfeld <Jason@zx2c4.com>
+# Copyright (C) 2016 Baptiste Jonglez <openwrt@bitsofnetworks.org>
+# Copyright (C) 2016-2017 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=amneziawg-tools
+PKG_VERSION:=1.0.20240213
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/amnezia-vpn/amneziawg-tools/archive/refs/tags/
+PKG_HASH:=4bde122630c9ddb1ec013c3e958f2c613b9eea56834674dda92fcb423c6f4d10
+
+PKG_MAINTAINER:=Amnezia Admin <admin@amnezia.org>
+PKG_LICENSE:=GPL-2.0
```

Comment:
This LICENSE is deprecated. 

Reference: https://spdx.org/licenses/ ("Deprecated License Identifiers")

You should use it with ``only`` or ``later``